src/unix/bsd/apple/mod.rs

```diff
@@ -231,6 +231,11 @@ impl ::Clone for sysdir_search_path_domain_mask_t {
 }
 
 s! {
+    pub struct ctl_info {
+        pub ctl_id: u32,
+        pub ctl_name: [::c_char; 96],
```

Comment:
The size here is too big for older rustc, we could nest the array to pass CI, see the below for example:
https://github.com/rust-lang/libc/blob/72fba0ecc76bf385d8f7b687fa8e46a1f6c0ae43/src/unix/bsd/apple/mod.rs#L833-L835

Reply:
Is there an option to compile differently for old and new `rustc`? I can image this workaround can cause some confusions when type checking.

Reply:
Maybe we could add cfg under https://github.com/rust-lang/libc/blob/master/build.rs.

Reply:
I add a cfg to compile `ctl_info` only for new `rustc`, please review. Thanks.